package-lock.json

```diff
@@ -5526,14 +5543,15 @@
       "dev": true
     },
     "fsevents": {
-      "version": "1.2.8",
-      "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.8.tgz",
-      "integrity": "sha512-tPvHgPGB7m40CZ68xqFGkKuzN+RnpGmSV+hgeKxhRpbxdqKXUFJGC3yonBOLzQBcJyGpdZFDfCsdOC2KFsXzeA==",
+      "version": "1.2.11",
```

Comment:
fsevents needed to be updated to 1.2.9 or higher for compatiliby w node 12 (i found the issue on fsevents repo)

since this is a dependency of a dependency (jest), i updated this by running `npm install fsevents@^1.2.9` and then `npm uninstall fsevents`

Reply:
you have to install and then uninstall?